okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -445,12 +453,21 @@ void stopIfNecessary() {
         try {
           socket.close();
         } catch (IOException e) {
-          log.log(Level.WARNING, "Failed closing socekt", e);
+          log.log(Level.WARNING, "Failed closing socket", e);
         }
       }
     }
   }
 
+  boolean mayHaveCreatedStream(int streamId) {
+    synchronized (lock) {
+      if (streamId >= 3 && streamId < nextStreamId && (streamId & 1) == 1) {
```

Comment:
Change this to just a return statement; no need for an if since you are returning a boolean.